frontend/viewer/README.md

```diff
@@ -13,10 +13,14 @@ for the english text 'Logout'. Then to make localization files run:
 ```bash
 pnpm run i18n:extract
 ```
-this will update the files under /src/locales/
-if you want you can then feed those files to an AI and it'll translate them for you.
+This will update the files under `/src/locales/`. If you want, you can then feed those files to an AI and it will translate them for you.
 
-#### advanced usage
+#### Adding a new language
+The `/src/locales/` folder contains one file per language, named using its language code (e.g. `es.json` for Spanish).
+To add a new language, copy `en.json`, rename it, translate the `"translation"` fields, and commit the changes.
+You must also update `frontend/viewer/lingui.config.ts` with the additional language code.
```

Comment:
I think if you do this step first, then run extract then ingui will generate the new file for you, no copying required.